lib/logcache/client.rb

```diff
@@ -46,6 +46,11 @@ def container_metrics(source_guid:, start_time:, end_time:, envelope_limit: DEFA
 
     def with_request_error_handling(_source_guid)
       tries ||= 3
+      start_time = Time.now
+
+      # convert to milliseconds to get more precise information
+      time_taken = (Time.now - start_time) * 1000
+      logger.info("Response time: #{time_taken.round(2)} ms")
```

Comment:
Is there a way to add the response time additionally in a parsable formation (json data like we do for vcap-request-id)?

Reply:
I can take a look